scania-digital-design-system/tegel
# feat(dropdown): prefix change

**Describe pull-request**  
Prefix change for Dropdown and DropdownV2 component

**Solving issue**  
Fixes: DTS-1636

**How to test**  
1. Go to the Storybook link below
2. Check if both Dropdown and DropdownV2 look and behave as versions in main branch
3. Check if the prefix is changed in all relevant places

**Suggested test steps**

**Additional context**  
:root selector is added for dropdown-v2
